execution_engine2.spec

```diff
@@ -149,12 +149,20 @@
 
         typedef structure {
             job_id parent_job_id;
-            list<job_id> children_job_ids;
+            list<job_id> child_job_ids;
```

Comment:
you could also make it even clearer by calling it `child_job_id_list`

Reply:
I've been scolded before for putting the type of the variable in the name of the variable.... So not sure if I should.

Reply:
Ha! Well, I don't have very strong feelings about it so if you don't want to provoke the ire of whoever scolded you before, leave it as-is. :)